tests/integration/authentication.test.js

```diff
@@ -31,8 +31,7 @@ describe('AUTHENTICATION ROUTES - POST /users/authenticate', () => {
   });
 
   it('should authenticate a user and set firstMobileConnection', async () => {
-    const momentToDate = sinon.stub(momentProto, 'toDate');
-    momentToDate.returns('2020-12-08T13:45:25.437Z');
+    UtilsMock.mockCurrentDate('2020-12-08T13:45:25.437Z');
```

Comment:
Pourquoi avoir transformé le stub en mock ? (c'est une vraie question car je suis pas hyper a laise sur ces notions)

Reply:
Je t'avoue ne pas être au point sur la différence entre stub et mock. D'après [ce lien](https://stackoverflow.com/questions/62675699/difference-between-fake-spy-stub-and-mock-of-sinon-library-sinon-fake-vs-spy), j'ai l'impression que le mock c'est comme un stub mais on définit en avance ce qu'on attend de la méthode plutot qu'apres avec les insertions pour le stub. 

Ici je ne sais pas pourquoi on a nommé le fichier utils Mock plutot que stub 🤔